documentation/homework/week1/src/components/todoListItem.js

```diff
@@ -0,0 +1,24 @@
+import React,{Component} from 'react';
+
+class TodoListItem extends Component {
+    render() {
+        const id = this.props.id;
+        const description = this.props.description;
+        const deadline = this.props.deadline;
+        const done = this.props.done ? '◉' : '◎';
+        return (
+            <div className="mainContent">
+                <div className="content">
+                    {description}
+                     --->
+                    {deadline}
+                    <span className='doneClick' onClick = {()=> this.props.handleDoneClick(id)}>{done}</span>
```

Comment:
For writing good function names, you need to use real and meaningful verbs for what your function is doing. `handle` is bad practice to name your function but instead use **checkTodo** or **toggleTodo**.